.github/CODEOWNERS

```diff
@@ -98,6 +98,7 @@
 /tox.ini @kevinlee12
 /scripts/ @kevinlee12
 /ubuntu_dockerfile @ezl-13 @kevinlee12
```

Comment:
Move this above around line 95 (in sorted order)

Reply:
done!